aii-ks/src/main/perl/ks.pm

```diff
@@ -692,13 +692,12 @@ sub log_action {
     }
 
     if ($consolelogging) {
-        push(@logactions, '# Make sure messages show up on the serial console',
-                          "tail -f $logfile > /dev/console &");
+        push(@logactions, "tee $logfile", "tee /dev/pts/0");
```

Comment:
Good suggestion! I didn't try, I will do it. Clearly if at the end this is just the matter of changin `console` to `pts/0`, it would be an easy change...